src/server/gql/schema/User.ts

```diff
@@ -0,0 +1,83 @@
+import { gql } from 'apollo-server-express'
+import { makeExecutableSchema } from 'graphql-tools'
+import { PrismaClient, User, Role } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+export default makeExecutableSchema({
+  typeDefs: gql`
+    type Query {
+      user(email: String!): User
+    }
+    type Mutation {
+      addUser(email: String!, givenName: String!, familyName: String!): User
+      assignRole(email: String!, role: String): User
+    }
+    type User {
+      id: Int
+      givenName: String!
+      familyName: String!
+      email: String!
+      roles: [String]
+    }
+  `,
+  resolvers: {
+    Query: {
```

Comment:
**Nitpick** **Question**

I might prefer a different organizational structure, and I'm curious what your opinion is. I'd prefer to put the typedefs and resolvers for top-level entities like `Query` and `Mutation` in their own file, then have a separate file for `User` which contains the typedefs and resolvers relating to users. The file with the object containing the `Query` resolvers could then import the `user` attribute resolver from the `User` file. What are your thoughts?